tests/e2e/env/setup.sh

```diff
@@ -176,6 +176,22 @@ cli wp plugin activate $DEV_TOOLS_DIR
 echo "Setting Jetpack blog_id"
 cli wp wcpay_dev set_blog_id $BLOG_ID
 
+if [[ ! ${SKIP_WC_SUBSCRIPTIONS_TESTS} ]]; then
+	echo "Install and activate the latest release of WooCommerce Subscriptions"
+	cd $E2E_ROOT/deps
+	LATEST_RELEASE=$(curl -H "Authorization: token $E2E_GH_TOKEN" -sL https://api.github.com/repos/$WC_SUBSCRIPTIONS_REPO/releases/latest | grep '"tag_name":' | cut -d'"' -f4)
+	curl -LJO -H "Authorization: token $E2E_GH_TOKEN" "https://github.com/$WC_SUBSCRIPTIONS_REPO/archive/$LATEST_RELEASE.zip"
+
+	unzip -qq woocommerce-subscriptions-$LATEST_RELEASE.zip
+	sudo mv woocommerce-subscriptions-$LATEST_RELEASE/* $E2E_ROOT/deps/woocommerce-subscriptions
```

Comment:
This `sudo` call might prompt for a password when running locally and it's not clear from the console which password should be used. Do you think we should add an `echo` to clarify that?

```
Install and activate the latest release of WooCommerce Subscriptions
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   177    0   177    0     0    264      0 --:--:-- --:--:-- --:--:--   264
100 2436k    0 2436k    0     0   877k      0 --:--:--  0:00:02 --:--:-- 2047k
curl: Saved to filename 'woocommerce-subscriptions-3.1.1.zip'
Password:
```

Reply:
I like that idea. What do you think about something like "Moving the unzipped plugin files. This may require your admin password" ?

Reply:
> What do you think about something like "Moving the unzipped plugin files. This may require your admin password" ?

💯 Sounds great!

Reply:
Great! Thanks for the feedback--added :D